Documentation/Configuration.rst

```diff
@@ -0,0 +1,88 @@
+Configuration
```

Comment:
This should go under the Developer's Guide.  It's pretty specific to doing development on processors.

Reply:
addressed in 36a919a by copying the config page into the dev page (both still slightly out of date, but it's a start)